python/tests/io/test_pyarrow.py

```diff
@@ -18,73 +18,19 @@
 
 import os
 import tempfile
-from typing import Any, List, Optional
```

Comment:
Can you move this refactor to a different PR? Then we can easily validate the refactor independently. For this PR, all that we should do is write tests in a separate test file.

Reply:
Sure. I reverted the refactoring to `test_pyarrow`. Will open a new PR soon